server/start.go

```diff
@@ -317,12 +317,6 @@ func startInProcess(svrCtx *server.Context, clientCtx client.Context, opts Start
 		return err
 	}
 
-	defer func() {
```

Comment:
how about [this](https://github.com/crypto-org-chain/ethermint/blob/363d814a585ab247bcce157a37e83da7bf0467d3/server/start.go#L248) close?

Reply:
good catch, removed that too.